test/mocha.opts

```diff
@@ -3,4 +3,5 @@
 --check-leaks
 --timeout 10000
 --require ./test/common
+--require ./test/promisePolyfill
```

Comment:
> It was necessary to include it for the documentation tests

Weird, I'd assume that it'd be enough to add it to bootstrap-unexpected-markdown.js. But never mind, let's just use it in the tests instead :)